src/gui/layout/qgslayoutlabelwidget.cpp

```diff
@@ -50,7 +50,7 @@ QgsLayoutLabelWidget::QgsLayoutLabelWidget( QgsLayoutItemLabel *label )
   connect( mMiddleRadioButton, &QRadioButton::clicked, this, &QgsLayoutLabelWidget::mMiddleRadioButton_clicked );
   setPanelTitle( tr( "Label Properties" ) );
 
-  mFontButton->setMode( QgsFontButton::ModeQFont );
```

Comment:
is ModeQFont used anywhere? can we remove it (or at least deprecate it ?) since it crashes on some MacOS versions?